bundles/org.openhab.transform.basicprofiles/README.md

```diff
@@ -27,13 +27,13 @@ Switch lightsStatus {
 
 The Generic Toggle Switch Profile is a specialization of the Generic Command Profile and toggles the State of a Switch Item whenever one of the specified events is triggered.
 
-### Configuration
+### Generic Toggle Switch Profile Configuration
```

Comment:
Why did you change that? Isn't this already clear by the structure (`##` denotes the profile, `###` the section within that profile)

Reply:
1. I followed the markdown warning that the headings should be unique.
2. This makes the anchor links better, but it's a very obscure benefit perhaps.  Without this, the links would look like `#configuration-1`, `#configuration-2`, etc.

Should I revert this?